HEP-event-capture/src/Dhis2Metadata.ts

```diff
@@ -0,0 +1,122 @@
+import fetch from "node-fetch";
```

Comment:
As we already discussed, we should check if it makes sense to re-use code between HEP-event-capture and HEP-data-entry.